client/src/main/java/org/spine3/change/Changes.java

```diff
@@ -35,10 +35,12 @@
 @SuppressWarnings("OverlyCoupledClass") /* ... because we want one utility class for all the Changes classes. */
 public class Changes {
 
-    private static final String PREVIOUS_VALUE = "previousValue";
-    private static final String NEW_VALUE = "newValue";
-    private static final String ERR_CANNOT_BE_EQUAL = "newValue cannot be equal to previousValue";
-    private static final String ERR_NEW_VALUE_CANNOT_BE_EMPTY = "newValue cannot be empty";
+    public interface ErrorMessage {
+        String PREVIOUS_VALUE = "previousValue";
+        String NEW_VALUE = "newValue";
+        String ERR_CANNOT_BE_EQUAL = "newValue cannot be equal to previousValue";
```

Comment:
Assuming we have these strings under the interface, I would suggest replacing the `ERR` prefix with meaningful starts. E.g. here just `VALUES_CANNOT_BE_QUEAL` would look good.